UI/Components/ServerComponent.cs

```diff
@@ -170,6 +170,7 @@ void connection_MessageReceived(object sender, MessageEventArgs e)
 
         private void ProcessMessage(String message, Connection clientConnection)
         {
+            var response = "";
```

Comment:
Would `string response = null;` make more sense here as a way to be more explicit, since we don't send the response back to the client unless its value is set?

Reply:
Sure. It's fundamentally the same thing, but if you're happier with that syntax then I'm fine with making that change.